code/modules/europa/planar_lighting/light_effect.dm

```diff
@@ -36,9 +36,12 @@
 	..(get_turf(holder))
 
 /obj/effect/light/Destroy()
-	moved_event.unregister(holder, src)
-	dir_set_event.unregister(holder, src)
-	destroyed_event.unregister(holder, src)
+	if(moved_event)
+		moved_event.unregister(holder, src)
+	if(dir_set_event)
+		dir_set_event.unregister(holder, src)
+	if(destroyed_event)
+		destroyed_event.unregister(holder, src)
```

Comment:
Dunno... I couldn't recreate the issues locally, so I haven't really been able to test if the problem in #553 is gone.